README.md

```diff
@@ -30,52 +30,53 @@ _Data is initially pulled from Stanford's system, but the user can update certai
 
 | Column Name | Type | Meaning | Notes |
 | - | - | - | - |
-| `SUNet` | string (primary key) | SUNet id | never shown to users, only used internally |
-| `first_name` * | string | first name |  |
-| `last_name` * | string| last name |  |
-| `email` * | string (email address) | user's email address | assumed initially to be SUNet@stanford.edu, but user can update |
+| `user_id` | string (primary key) | uuid | |
+| `dining_hall_id` | string (foreign key) | uuid - dining hall the user is currently at |
```

Comment:
I'm thinking this is where we'll store which dining hall the user says they're at in the initial dropdown. This will keep changing as the user updates where they are, and it will be `null` if the user indicates that they're not in a dining hall.

Reply:
is there an advantage to using a uuid over SUNet? (ignoring the above privacy question for now) -- I don't super care, so whatever works

good call on the dining hall, duh

Reply:
For our number of users it truly would not matter lol but generally, you want to avoid the risk of certain areas of the db getting hit harder (i.e. "hot spots"). E.g. if there are a ton of sunet ids that start with "ab" or something, this section of the db will get hit harder with requests than another section like ids that start with "zu." So there's a bit of a performance risk at scale. Randomly generated UUIDs help prevent this risk.